R/linter_tag_docs.R

```diff
@@ -80,3 +80,12 @@ NULL
 #' @evalRd rd_linters("package_development")
 #' @seealso [linters] for a complete list of linters available in lintr.
 NULL
+
+#' Deprecated linters
+#' @name deprecated_linters
+#' @description
+#' Linters that are deprecated and provided for backwards compatiblility only.
+#' These linters will be excluded from `linters_with_tags()` by default.
+#' @evalRd rd_linters("deprecated")
+#' @seealso [linters] for a complete list of linters available in lintr.
+NULL
```

Comment:
now i see why `?deprecated_linters` wasn't being generated in #1109 -- didn't have the `NULL` anchor! 🤦